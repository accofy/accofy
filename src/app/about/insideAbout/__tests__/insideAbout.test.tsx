import '@testing-library/jest-dom';

import InsideAboutPage from '@/app/about/insideAbout/page';
import AboutLayout from '@/app/about/layout';
import renderWithProviders from '@/utils/renderWithProviders';

describe('Inside About route', () => {
  it('snapshot testing', () => {
    const { container } = renderWithProviders(
      <AboutLayout>
        <InsideAboutPage />
      </AboutLayout>,
    );
    expect(container).toMatchSnapshot();
  });
});
