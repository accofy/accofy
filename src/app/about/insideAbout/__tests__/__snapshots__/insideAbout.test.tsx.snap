// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inside About route snapshot testing 1`] = `
<div>
  <section
    class="flex min-h-screen flex-col items-center justify-center gap-6 bg-black text-white"
  >
    About layout
    <div
      class="flex flex-col items-center justify-center gap-5 rounded-lg border border-dashed border-gray-400 p-40 text-5xl"
    >
      <p>
        Inside About page
      </p>
      <a
        class="rounded-md border border-white/10 px-3 py-3  text-xl font-medium  transition-all duration-300 ease-in-out hover:border-white/5 hover:bg-white/10"
        href="/about"
      >
        Go Back
      </a>
    </div>
  </section>
</div>
`;
